### PR TITLE
#88 Implement List of Slack Channels from Slack-archive service 

### PR DESCRIPTION
**Pull request by petrokramar**
Connect to #88

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -7,16 +7,16 @@ import { HttpModule } from '@angular/http';
...
-import { AllUsersTableComponent } from './components/all-users-table/all-users-table.component';
+import { AllUsersTableComponent } from './components/gamification/all-users-table/all-users-table.component';
 import { GamificationService } from './service/gamification.service';
 import { UserService } from './service/user.service';
-import { UserDetailsTableComponent } from './components/user-details-table/user-details-table.component';
+import { UserDetailsTableComponent } from './components/gamification/user-details-table/user-details-table.component';
 import { ScrollDirective } from './directives/scroll.directive';
 import { GoTopComponent } from './components/go-to-top/go-to-top.component';
-import { ActiveKeepersComponent } from './components/active-keepers/active-keepers.component';
+import { ActiveKeepersComponent } from './components/gamification/active-keepers/active-keepers.component';
```
Comment (adaleorion) [inline src/app/app.module.ts]: I think keepers is different service. 
So could you change name root folder to keeper

File: src/app/app.module.ts [hunk 2]
```diff
@@ -7,16 +7,16 @@ import { HttpModule } from '@angular/http';
...
 import { KeepersService } from './service/keepers.service';
 import { TeamsService } from './service/teams.service';
-import { TeamsComponent } from './components/teams/teams.component';
+import { TeamsComponent } from './components/gamification/teams/teams.component';
```
Comment (adaleorion) [inline src/app/app.module.ts]: I think teams is different service. 
So could you change root folder name to team

Comment (petrokramar) [reply src/app/app.module.ts]: Done

File: src/app/app.module.ts (hunk 1 above)
Comment (petrokramar) [reply src/app/app.module.ts]: Done

**Review by adaleorion — APPROVED** [review]
cool, thanks

**Pull request merged**